pyflakes/checker.py

```diff
@@ -1139,15 +1146,15 @@ def IMPORTFROM(self, node):
                 # Only Python 2, local import * is a SyntaxWarning
                 if not PY2 and not isinstance(self.scope, ModuleScope):
                     self.report(messages.ImportStarNotPermitted,
```

Comment:
Just noting for clarity the `if module.endswith('.')` is only in `ImportationFrom` at the moment, and not yet in `StarImportation` or this `self.report` invocation.

Im flexible regarding implementation; im not sure about the UI.
`.*`, `..*`, `...*` are all rather ambiguous regarding how many levels were used. i.e. `.*` and `..*` could   be reasonable interpreted by different people to mean exactly the same thing.

One way to avoid this ambiguity is to change the message so it uses `from .. import *` instead of `..*`

Of course the line of code has the real import, and the message doesnt need to precisely describe the import, so this UI problem is quite minor.